cdc/model/schema_storage.go

```diff
@@ -104,7 +104,6 @@ func WrapTableInfo(schemaID int64, schemaName string, version uint64, info *mode
 		TableName: TableName{
 			Schema:      schemaName,
 			Table:       info.Name.O,
-			TableID:     info.ID,
```

Comment:
Please remove redunant comments in [TableInfo.TableName](https://github.com/pingcap/tiflow/blob/master/cdc/model/schema_storage.go#L40-L49).